ts/Core/Chart/MapChart.ts

```diff
@@ -0,0 +1,218 @@
+            },
+            userOptions, // user's options
+
+            { // forced options
+                chart: {
+                    inverted: false,
+                    alignTicks: false
+                }
+            }
+        );
+
+        userOptions.series = seriesOptions;
+
+        super.init(userOptions, callback);
+    }
+}
+
+/* eslint-disable valid-jsdoc */
+
+/**
+ * The factory function for creating new map charts. Creates a new {@link
+ * Highcharts.MapChart|MapChart} object with different default options than the
+ * basic Chart.
+ *
+ * @requires modules/map
+ *
+ * @function Highcharts.mapChart
+ *
+ * @param {string|Highcharts.HTMLDOMElement} [renderTo]
+ * The DOM element to render to, or its id.
+ *
+ * @param {Highcharts.Options} options
+ * The chart options structure as described in the
+ * [options reference](https://api.highcharts.com/highstock).
+ *
+ * @param {Highcharts.ChartCallbackFunction} [callback]
+ * A function to execute when the chart object is finished loading and
+ * rendering. In most cases the chart is built in one thread, but in
+ * Internet Explorer version 8 or less the chart is sometimes initialized
+ * before the document is ready, and in these cases the chart object will
+ * not be finished synchronously. As a consequence, code that relies on the
+ * newly built Chart object should always run in the callback. Defining a
+ * [chart.events.load](https://api.highcharts.com/highstock/chart.events.load)
+ * handler is equivalent.
+ *
+ * @return {Highcharts.MapChart}
+ * The chart object.
+ */
+function mapChart(
+    a: (string|HTMLDOMElement|Highcharts.Options),
+    b?: (Chart.CallbackFunction|Highcharts.Options),
+    c?: Chart.CallbackFunction
+): MapChart {
+    return new MapChart(a as any, b as any, c);
+}
+
+/**
+ * Utility for reading SVG paths directly.
+ *
+ * @requires modules/map
+ *
+ * @function Highcharts.splitPath
+ *
+ * @param {string|Array<string|number>} path
+ *
+ * @return {Highcharts.SVGPathArray}
+ */
+function splitPath(
+    path: string|Array<string|number>
+): SVGPath {
+    let arr: Array<string|number>;
+
+    if (typeof path === 'string') {
+        path = path
+            // Move letters apart
+            .replace(/([A-Za-z])/g, ' $1 ')
+            // Trim
+            .replace(/^\s*/, '').replace(/\s*$/, '');
+
+        // Split on spaces and commas. The semicolon is bogus, designed to
+        // circumvent string replacement in the pre-v7 assembler that built
+        // specific styled mode files.
+        const split = path.split(/[ ,;]+/);
+
+        arr = split.map((item): (number|string) => {
+            if (!/[A-za-z]/.test(item)) {
+                return parseFloat(item);
+            }
+            return item;
+        });
+    } else {
+        arr = path;
+    }
+
+    return SVGRenderer.prototype.pathToSegments(arr);
+}
+
+const mapModule = {
+    maps,
+    mapChart,
+    MapChart,
+    splitPath
+};
```

Comment:
I have an idea, that is similar to static functions/props. You could create a namespace with the name `MapChart` and put all the functions/props into this namespace as exports. Then TypeScript merges it with the class and you just need to export MapChart class. Functions/props can be accessed like `MapChart.maps` etc.